src/index.html

```diff
@@ -2,15 +2,54 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="searchContainer--big searchContainer"
+      action="/"
+      method="post"
+      data-qa="big"
+    >
+      <img
+        class="searchIcon--big"
+        src="images/Search.svg"
+        alt="magnifying glass"
+      >
+      <div class="inputWrapper">
```

Comment:
do you need this wrapper?

Reply:
Yes, we need this wrapper.
It allows input to sit next to img which floats to the left and input takes all available width.

Reply:
Wrapper removed